### PR TITLE
Fix _entities

### PR DESCRIPTION
**Pull request by adam-urbanczyk**
Fix iteration over entities and avoid hash collisions. This should solve intermittent test failures.

### CODE REVIEW

File: cadquery/occ_impl/shapes.py [hunk 1]
```diff
@@ -747,18 +751,21 @@ def ShapeType(self) -> Shapes:
...
 
         explorer = TopExp_Explorer(self.wrapped, inverse_shape_LUT[topo_type])
 
         while explorer.More():
             item = explorer.Current()
-            out[
-                item.HashCode(HASH_CODE_MAX)
-            ] = item  # needed to avoid pseudo-duplicate entities
+
+            # needed to avoid pseudo-duplicate entities
```
Comment (adam-urbanczyk) [inline cadquery/occ_impl/shapes.py:762]: OCP does not wrap inner iterators (including `MapOfShape`), so this kind of construction is needed here.

File: appveyor.yml [hunk 2]
```diff
@@ -28,7 +28,6 @@ init:
 install:
     - mamba env create -f environment.yml
     - conda activate cadquery
-    - pip install git+https://github.com/CadQuery/OCP-stubs.git@7.6.3
```
Comment (jmwright) [inline appveyor.yml:31]: The stubs really aren't needed anymore?

**Review by jmwright — APPROVED** [review]

File: appveyor.yml (hunk 2 above)
Comment (adam-urbanczyk) [reply appveyor.yml:31]: They already specified in the environment file

Comment (jmwright) [reply appveyor.yml:31]: Ok, thanks.

**Review by lorenzncode — APPROVED** [review]
Given the test failures are intermittent I was unable to reproduce the issue with the old method to compare with the fix.  I tested the PR branch and did not find any problem with the new implementation.  Good find if this solves the intermittent test failures.

**Pull request merged**